main.bal

```diff
@@ -51,36 +54,17 @@ service class Service {
         int stringLength = <int>self.lengthOne + <int>self.lengthTwo;
         if (self.expectedLen == stringLength) {
```

Comment:
Dose this print `Content Length Does not Match the Message Length` for the first message everytime ? for the non matching case we don't need to print anything, we simply have to return and wait until we get the message